plugins/cost-insights/src/components/CostGrowth/CostGrowthIndicator.tsx

```diff
@@ -25,7 +25,10 @@ import { useCostGrowthStyles as useStyles } from '../../utils/styles';
 
 export type CostGrowthIndicatorProps = TypographyProps & {
   change: ChangeStatistic;
-  formatter?: (change: ChangeStatistic) => Maybe<string>;
+  formatter?: (
+    change: ChangeStatistic,
+    returnAbsoluteValue: boolean,
```

Comment:
Could we make this `options: { absolute: boolean }` instead? That's a pattern we use a lot (naked booleans are not great from a usability/readability standpoint), and then we can add more things to that object in the future.